CRM/Financial/BAO/FinancialAccount.php

```diff
@@ -246,4 +246,48 @@ public static function getFinancialAccountForFinancialTypeByRelationship($financ
     return Civi::$statics[__CLASS__]['entity_financial_account'][$financialTypeID][$relationTypeId];
   }
 
+  /**
+   * check if financial type has Deferred Revenue Account is relationship
+   * with Financial Account
+   *
+   * @param $financialTypeId Integer
+   *
+   * @return string
+   *
+   */
+  public static function validateFinancialType($financialTypeId, $entityID = NULL, $entity = NULL) {
+    if (!$financialTypeId) {
```

Comment:
Hmm - it still feels like why would you call this function if you didn't have a financial type to validate. It just feels like it's encouraging bad coding to do this.